geo.txt

```diff
@@ -56,3 +56,1703 @@ Phys 91(11):957-962
+Chem Phys 131(4) 044303/1-044303/5
+doi: 10.1063/1.3186756
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+# O-interior
+
+OCO SCO 3.9862(10)
+θ1      118.0 (44) 
+θ2      135.8 (24)
+
+# MW
+
+OCO SCO 3.9816(16)
+θ1      108.8(46)
+θ2      140.7(46)
+
+# S-interior
+
+OCO SCO 3.5545 (10)
+θ1      69.7 (34)
+θ2      78.7 (25)
+
+# θ1: Angle between Rcm and the O=C=O axis (measured from the inner O end)
+# θ2: Angle between Rcm and the O=C=S axis (measured from the S end).
+
+NAME = C2S4
+# Point Group: D2d
+# Shape: Linear --- Linear
+# Bond lengths: r0     (angstroms)
+# Source paper: Rezaei M, Norooz Oliaee J, Moazzen-Ahmadi N, McKellar ARW (2011) Spectroscopic observation and
+structure of CS2 dimer. J Chem Phys 134(14):144306/1-144306/5
+doi:10.1063/1.3578177
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C = C     3.539(7)
+C = C = S 90
+
+NAME = C3Ge2
+# Point Group: Dooh 
+# Shape: Linear
+# Bond lengths: re (angstroms)
+# Source paper: Thorwirth S, Lutter V, Javed AJ, Gauss J, Giesen TF (2016) Gas-phase spectroscopic detection and structural
+elucidation of carbon-rich group 14 binary clusters: Linear GeC3Ge. J Phys Chem A 120(2):254-259
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C ≡ Ge 1.7695(1)
+C = C  1.2893
+
+# C = C 1.2893 assumed at the CCSD(T)/cc-pwCVQZ value
+
+NAME = C3Pd
+# Point Group: Coov 
+# Shape: Linear
+# Comment: The molecule was found to be linear or quasilinear.
+# Bond lengths: r0 (angstroms)
+# Source paper: Bittner DM, Zaleski DP, Tew DP, Walker NR, Legon AC (2016) Highly unsaturated platinum and palladium
+carbenes PtC3 and PdC3 isolated and characterized in the gas phase. Angew Chem 128(11):3832-3835; Angew
+Chem Int Ed Engl 55(11):3768-3771
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Pd   = C(1) 1.79898(4)
+C(1) = C(2) 1.3009
+C(2) = C(3) 1.2789
+
+# C(1) = C(2) 1.3009 and C(2) = C(3) 1.2789 fixed to the value calculated at the level CCSD(T)_ae/aug-cc-pwCV5Z-PP(for Pd) and corrected for r0 ‒ re
+difference assumed to that for PtC3. 
+
+NAME = C3Pt
+# Point Group: Coov 
+# Shape: Linear
+# Comment: The molecule was found to be linear or quasilinear.
+# Bond lengths: r0 (angstroms)
+# Source paper: Bittner DM, Zaleski DP, Tew DP, Walker NR, Legon AC (2016) Highly unsaturated platinum and palladium
+carbenes PtC3 and PdC3 isolated and characterized in the gas phase. Angew Chem 128(11):3832-3835; Angew
+Chem Int Ed Engl 55(11):3768-3771
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Pt   = C(1) 1.7315(14)
+C(1) = C(2) 1.2993(19)
+C(2) = C(3) 1.2759(11)
+
+NAME = C4HNO 2012
+# Point Group: Coov 
+# Shape: Linear
+# Bond lengths: r0 (angstroms)
+# Source paper: Kang L, Novick SE (2012) The microwave spectra of the weakly bound complex between carbon monoxide and
+cyanoacetylene, OC H–C≡C–C≡N. J Mol Spectrosc 276-277:10-13
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C(4)...H        2.65315(27)
+C(4)...D        2.64962(27)
+OC4    HC1C2C3N 6.2031
```

Comment:
Maybe the following would work:
```
OC(1) C3HN 6.2031
C(1)  H    2.65315(27)
```
But the only problem is that it's not obvious that the H is the atom closest to CO.

Also, C...H  and C...D belong in separate sections. D is an isotope of H.
```